.circleci/config.yml

```diff
@@ -0,0 +1,91 @@
+version: 2
+jobs:
+  build:
+
+    # Primary command image where all commands run
+    docker:
+      - image: circleci/ruby:2.4.2-node-browsers
+        environment:
+          RAILS_ENV: test
+          ORANGELIGHT_HOST: localhost
+          ORANGELIGHT_USER: postgres
+
+      # Service container image available at 'host: localhost'
+      - image: circleci/postgres:9.6-alpine    
```

Comment:
Can we switch this to postgres 10?

Reply:
I remembered that orangelight uses in production postgres 9.6. If it won't create a conflict I can update the entry to postgres:10.